api/src/main/java/com/codeforcommunity/rest/ApiRouter.java

```diff
@@ -88,45 +99,86 @@ public Router initializeRouter(Vertx vertx) {
      * afterRoute.handler(this::handleAfter);
      */
 
-    Route signUpRoute = router.route("/signup");
+    // USER CRUD
+
+    // Create User : Non-Protected POST method
+    Route signUpRoute = router.post("/signup");
     signUpRoute.handler(this::handleSignUp);
 
+    // Should any authorized user be able to see the information of another user?
```

Comment:
There are some good questions in these comments but really these should be asked why defining an api specification not in the code. Ideally you should have an api-spec document and work out these implementation details with the team. If you want to talk more about the design tradeoffs you can slack me about it